apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/TableEditor.tsx

```diff
@@ -60,8 +61,13 @@ const TableEditor = ({
   const isNewRecord = isUndefined(table)
 
   const realtimeEnabled = useIsFeatureEnabled('realtime:all')
-
-  const enumTypes = meta.types.list((type: PostgresType) => !EXCLUDED_SCHEMAS.includes(type.schema))
+  const { data: types } = usePostgresTypesQuery({
+    projectRef: project?.ref,
+    connectionString: project?.connectionString,
+  })
+  const enumTypes = (types ?? []).filter(
+    (type) => !EXCLUDED_SCHEMAS.filter((x) => x !== 'extensions').includes(type.schema)
```

Comment:
small perf nit: could move the `EXCLUDED_SCHEMAS.filter((x) => x !== 'extensions')` part out of this `.filter()` into its own const (maybe something like `EXCLUDED_SCHEMAS_WITHOUT_EXTENSIONS`) above the component. That way it we can avoid some nested loops :)

Reply:
gotcha!